tests/unit/test_utils.py

```diff
@@ -31,17 +31,18 @@ class TestEnum(Enum):
 def test__filter_out_none_values_recursively() -> None:  # Copypasted from client
     assert filter_out_none_values_recursively({'k1': 'v1'}) == {'k1': 'v1'}
     assert filter_out_none_values_recursively({'k1': None}) == {}
-    assert filter_out_none_values_recursively({'k1': 'v1', 'k2': None, 'k3': {'k4': 'v4', 'k5': None}, 'k6': {'k7': None}}) \
-        == {'k1': 'v1', 'k3': {'k4': 'v4'}}
+    assert filter_out_none_values_recursively(
+        {'k1': 'v1', 'k2': None, 'k3': {'k4': 'v4', 'k5': None}, 'k6': {'k7': None}}
+    ) == {'k1': 'v1', 'k3': {'k4': 'v4'}}
 
 
-def test__filter_out_none_values_recursively_internal() -> None:  # Copypasted from client
-    assert _filter_out_none_values_recursively_internal({}) == {}
-    assert _filter_out_none_values_recursively_internal({'k1': {}}) == {}
-    assert _filter_out_none_values_recursively_internal({}, False) == {}
-    assert _filter_out_none_values_recursively_internal({'k1': {}}, False) == {'k1': {}}
-    assert _filter_out_none_values_recursively_internal({}, True) is None
-    assert _filter_out_none_values_recursively_internal({'k1': {}}, True) is None
+def test_filter_out_none_values_recursively_internal() -> None:  # Copypasted from client
```

Comment:
Hmm, I'm thinking this lovely comment was created before we had the shared package, can you check if it's still correct to have something "copypasted"?